server/app/controllers/api/v1/experiences_controller.rb

```diff
@@ -47,7 +49,12 @@ def experience_params
   end
 
   def set_api_user
-    @api_user = ApiUser.find(params[:api_user_id])
+    if !params[:api_user_id].nil?
```

Comment:
Rails comes with handy string helpers:

```suggestion
    if params[:api_user_id].present?
```

The benefit of this is that we can avoid negated logic conditionals. "if not nil" is a tiny bit harder to process for some human readers.

Reply:
Agree and updated